inst/EpiModel2/SEIRModuleExtension.R

```diff
@@ -96,7 +96,7 @@ param <- param.net(inf.prob = 0.5, act.rate = 2, ei.rate = 0.01, ir.rate = 0.005
 init <- init.net(i.num = 10)
 ## If any user defined functions, type must equal NULL
 control <- control.net(type = NULL, nsteps = 100, nsims = 1, infection.FUN = infect2,
-                       progress.FUN = progress)
+                       progress.FUN = progress, prevalence.FUN = prevalence.net)
```

Comment:
But prevalence is a module that is in the special class such that it does not need to be specified explicitly, right? Check what is in your migration doc. Shouldn't it be handled like the network resimulation module?

Reply:
@andsv2 This prevalence fx still explicitly defined here. Why?